### ISSUE TITLE
Not compatible with VoiceOver

### ISSUE THREAD

**Issue by matteosistisette** [hammerjs/hammer.js#1101]
If VoiceOver is enabled, this plugin doesn't work (at least, I tried the demo).

The expected behavior is that, when voiceover is enabled (which captures swipe gestures), swipe gestures should be replaced by two- or three-finger swipe. That is, a two- or three-finger swipe should do what normally a single-finger swipe does. That's the general convention.
